theme_graphene/views/customizations.xml

```diff
@@ -109,6 +113,15 @@
     <xpath expr="//div[hasclass('carousel-inner')]/div[3]" position="attributes">
         <attribute name="style" add="background-position: 50% 100%;" separator=";"/>
     </xpath>
+    <xpath expr="//div[hasclass('o_we_bg_filter')]" position="attributes">
+        <attribute name="class" add="bg-white-25" remove="bg-black-25" separator=" "/>
```

Comment:
I think that adding `bg-white-25` it's not necessary ;) 

| this PR | Master |
|--------|--------|
| <img width="1187" alt="image" src="https://github.com/user-attachments/assets/053a83a5-68e9-41b1-8823-3218deca222e"> | <img width="1192" alt="image" src="https://github.com/user-attachments/assets/c9582d92-a88c-48a2-a627-c12cd2b01af6"> |